norse/torch/module/receptive_field.py

```diff
@@ -179,12 +234,32 @@ def __init__(
         optimize_ratios: bool = True,
         optimize_x: bool = True,
         optimize_y: bool = True,
+        optimize_log: bool = False,
```

Comment:
Again, isn't the sensible default `True`?

Reply:
It's true that setting it to true might make more sense (leading to a more stable training).